src/main/java/com/eprosima/fastdds/idl/grammar/Context.java

```diff
@@ -298,7 +298,7 @@ public String getPackageUnder()
     //! Cache the first exception.
     private com.eprosima.idl.parser.tree.Exception m_firstexception = null;
 
-    // TODO Lleva la cuenta de generación de nuevos nombres.
+    // TODO Lleva la cuenta de generacion de nuevos nombres.
```

Comment:
I'd rather default to English actually. What about?
```suggestion
    // TODO Keeps new names generation count.
```

Reply:
You are actually correct. I changed the comment and some others in the file to English. Please re-check if the change preserves the meaning (I'm not entirely sure, since I'd expect an imperative form after a TODO).